modules/serverless-functions-all-values-in-configmap-to-env-variables.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-all-values-in-configmap-to-env-variables_{context}"]
+= Setting environment variables from all values defined in a config map
+
+. Open the `func.yaml` file for your function.
+
+. For every config map, for which you want to import all key-value pairs as environment variables, add the following YAML to the `envs` section:
```

Comment:
s/map/map
The phrase _from which you want to import.._ isn't parenthetical; it is required to define which config maps you are talking about.

Reply:
@mburke5678 What's that about s/map/map?

How do you suggest I phrase the second sentence? What I'm trying to say here is equivalent to this:
"Let us say that you have a number of config maps. There is a subset of them such that for each member of the subset you want to import all key-value pairs of that member as environment variables. To achieve that, for each such member add the following YAML to the `envs` section:"

Reply:
Sorry! My mistake. Just remove the comma and you should be fine. 

_For every config map for which you want to import all key-value pairs as environment variables, add the following YAML to the `envs` section:_

Reply:
@mburke5678 Np and done!